lib/core/source/auth_remote_source.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter_template/core/model/service/auth_models.dart';
+import 'package:flutter_template/core/model/service/service_response.dart';
+import 'package:flutter_template/core/source/common/http_service.dart';
+
+class AuthRemoteSource {
+  final HttpServiceDio _httpService;
+
+  AuthRemoteSource({required HttpServiceDio httpService})
+      : _httpService = httpService;
+
+  Future<SignInResponse> signIn(String email, String password) async {
+    final response = await _httpService.post('authaccount/login',
```

Comment:
maybe static variables for the routes?

Reply:
I didn't use it because it's an example, but it makes sense!